addons/l10n_fr_certification/models/account_closure.py

```diff
@@ -0,0 +1,153 @@
+from datetime import datetime, timedelta
+
+from openerp import models, api, fields
+from openerp.fields import Datetime as FieldDateTime
+from openerp.tools.translate import _
+from openerp.exceptions import UserError
+
+WRITE_MSG = _('Sale Closings are not meant to be written or deleted under any circumstances.')
+
+
+class AccountClosure(models.Model):
+    _name = 'account.sale.closure'
```

Comment:
Moreover, don't you want to make a different closure per journal? what happens if you create a new sale journal? is it suppose to be allowed or not?